### PR TITLE
New Grants Recontribute Email

### LINKED ISSUE #6646: As a Gitcoin Grants Contributor, I'd like an email to kick off Round 6 so that I can one-click re-contribute to all the grants from last round.
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a Gitcoin Grants Contributor, I'd like an email to kick off Round 6 so that I can one-click re-contribute to all the grants from last round.

### Why Is this Needed

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:
to increase contribution counts

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*:
feature

### Current Behavior
[comment]: # (Describe what actually happened.)
no email

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
user receives an email with all of the grants they've previously funded, and a link to add them all to cart again.
- management command that can kick off the email
- spliced emial that looks good in browser and major mail clients

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

### Data Requirements
[comment]: # (How will we measure the success of this feature? What kind of tracking is needed for this feature (clicks, impressions, flag)?)

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)
https://www.figma.com/file/6OqyMtEfRU4vDMlZXO6Y3l/Gitcoin-Grants-Experience-2020?node-id=119%3A981

### PR DESCRIPTION
**Pull request by sebastiantf**
##### Description
This PR builds a new email to kick off a new grants round for Grants contributors to one-click re-contribute to all the grants from last round

##### Refers/Fixes
Fixes #6646 

##### Screenshots
**Gmail Web:**

grant_recontribute_email_gmail_web

**Gmail Mobile:**

grant_recontribute_email_gmail-mobile

**Web Preview:**

grant_recontribute_email_web_preview

### CODE REVIEW

**Comment by owocki** [comment]
looks good :)

**Comment by owocki** [comment]
yay thank you!

**Comment by sebastiantf** [comment]
The link for the fund grants button isn't yet assigned: https://github.com/gitcoinco/web/issues/6646#issuecomment-639540236

**Comment by PixelantDesign** [comment]
Click here to set payout address...could we use the blue link color like the section below?

**Comment by sebastiantf** [comment]
> Click here to set payout address...could we use the blue link color like the section below?

Yeah. Will try doing that.

**Comment by sebastiantf** [comment]
@PixelantDesign This is how it looks with the same blue color as in the footer. Is this fine?

Screenshot 2020-06-06 at 11 17 53 AM

**Comment by owocki** [comment]
looks good to me! yess

**Comment by PixelantDesign** [comment]
yes looks good thank you!

File: app/marketing/management/commands/grant_recontribute_email.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        
+        # Round 5: 3/23/2020 — 4/7/2020
+        prev_round_start = (2020, 3, 23)
+        prev_round_end = (2020, 4, 7)
+
+        # Round 6: 6/15/2020 — 6/29/2020
+        next_round = 6
+        next_round_start = (2020, 6, 15)
+        next_round_end = (2020, 6, 29)
+        match_pool = '250k'
```
[the remark this answers is not in the document]
Comment (sebastiantf) [reply app/marketing/management/commands/grant_recontribute_email.py]: Done in https://github.com/gitcoinco/web/pull/6777/commits/b794c2d90e91284c7eda449f3f252e78c63cbcaa

**Comment by sebastiantf** [comment]
> Click here to set payout address...could we use the blue link color like the section below?

Done in https://github.com/gitcoinco/web/pull/6777/commits/7a4ef6cf6a9c4551cf4794cf28297b8cb42c3fae

**Comment by sebastiantf** [comment]
> hey @sebastiantf can you make the 'fund grants' go to https://gitcoin.co/grants/cart/bulk-add/686,204 where 686,204 is a comma delimited list of grants to add to cart ? thanks

Done in https://github.com/gitcoinco/web/pull/6777/commits/d387cfb20b2858741eac506ba55a635220e7adbb

**Comment by owocki** [comment]
@sebastiantf can u point this PR at `grants-round-6-integration` branch? that is the integration branch for grants round 6

**Comment by owocki** [comment]
@sebastiantf awesome!

**Pull request merged**